examples/shadow/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/at-wat/mqtt-go"
+	"github.com/seqsense/aws-iot-device-sdk-go/v4"
+	"github.com/seqsense/aws-iot-device-sdk-go/v4/shadow"
+)
+
+func main() {
```

Comment:
I think we should have some documentation about how to run this example. The user needs to place `root-CA.crt`, `certificate.pem.crt`, `private.pem.key` in the right place, specify aws iot hostname as 1st arg, and replace the occurences of "sample" (thingName) with the right one.

Reply:
Added them to usage help output.

Reply:
Other examples are fixed in https://github.com/seqsense/aws-iot-device-sdk-go/pull/128

Reply:
Thanks! Much easier to launch the example 👍